org.omg.sysml.xtext/src/org/omg/sysml/xtext/util/SysMLRepositoryLoadUtil.java

```diff
@@ -0,0 +1,170 @@
+ */
+package org.omg.sysml.xtext.util;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.OptionGroup;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.omg.kerml.xtext.KerMLStandaloneSetup;
+import org.omg.sysml.ApiException;
+import org.omg.sysml.util.SysMLUtil;
+import org.omg.sysml.util.repository.EObjectUUIDTracker;
+import org.omg.sysml.util.repository.APIModel;
+import org.omg.sysml.util.repository.EMFModelDelta;
+import org.omg.sysml.util.repository.ProjectRepository;
+import org.omg.sysml.util.repository.Revision;
+import org.omg.sysml.util.repository.RemoteProject;
+import org.omg.sysml.util.repository.RemoteProject.RemoteBranch;
+import org.omg.sysml.util.repository.EMFModelRefresher;
+import org.omg.sysml.xmi.SysMLxStandaloneSetup;
+import org.omg.sysml.xtext.SysMLStandaloneSetup;
+
+public class SysMLRepositoryLoadUtil extends SysMLUtil {
+	
+	public static void main(String[] args) throws ParseException, UnsupportedOperationException, ApiException {
+		SysMLRepositoryLoadUtil sysMLRepositoryLoadUtil = createUsingArgs(args);
+		
+		KerMLStandaloneSetup.doSetup();
+		SysMLStandaloneSetup.doSetup();
+		SysMLxStandaloneSetup.doSetup();
+		
+		sysMLRepositoryLoadUtil.load();
+	}
+	
+	public static SysMLRepositoryLoadUtil createUsingArgs(String... args) throws ParseException {
+		var localLibrary = new Option("l", "library", true, "Path to local library");
+		localLibrary.setRequired(true);
+		
+		var repositoryOption = new Option("b", "base", true, "Repository url. E.g: http://localhost:9000");
+		repositoryOption.setRequired(false);
+		
+		OptionGroup projectOption = new OptionGroup();
+		projectOption.setRequired(true);
+		var projectNameOption = new Option("n", "name", true, "Project's name in the repository.");
+		var projectIdOption = new Option("id", "id", true, "Project's id in the repository.");
+		
+		projectOption.addOption(projectNameOption);
+		projectOption.addOption(projectIdOption);
+		
+		var branchOption = new Option("br", "branch", true, "Branch's name in the project");
+		
+		var targetOption = new Option("t", "target", true, "Location where the project is loaded");
+		targetOption.setRequired(true);
+		
+		Options options = new Options()
+				.addOption(repositoryOption)
+				.addOptionGroup(projectOption)
+				.addOption(branchOption)
+				.addOption(targetOption)
+				.addOption(localLibrary);
+		
+		
+		CommandLineParser parser = new DefaultParser(false);
+		
+		CommandLine cli = parser.parse(options, args);
+		
+		return new SysMLRepositoryLoadUtil(
+					cli.hasOption(repositoryOption)? cli.getOptionValue(repositoryOption) : "http://localhost:9000",

```

Comment:
This URL should be kept consistent with other base paths, e.g. the one in ApiElementProcessingFacade. The best would be if a single constant is used throughout the codebase.